pom.xml

```diff
@@ -83,7 +84,12 @@
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
         <version>4.13.2</version>
-        <scope>test</scope>
```

Comment:
Removed this here from the `<dependencyManagement>` because specifying the scope there seems to be discouraged, see https://stackoverflow.com/q/15221299

All the modules of this project using JUnit as test dependency already set the scope anyways apparently